dlt/common/data_writers/writers.py

```diff
@@ -23,6 +23,12 @@ class TFileFormatSpec:
     supports_compression: bool = False
 
 
+class DataWriterMetrics(NamedTuple):
+    file_path: str
+    items_count: int
+    file_size: int
```

Comment:
maybe column count? but that is not really important tbh.

Reply:
I plan to add elapsed time (start stop). Column count is not known at this moment

Reply:
not during extract. but it is known during normalize. you can however get the column count from the relevant schema...

Reply:
yes elapsed would be cool too!